_layouts/robot.html

```diff
@@ -56,5 +62,47 @@
     </div>
     {% endif %}
 
+<div style="display: block; margin-left: auto; margin-right: auto;">
```

Comment:
This does not appear to be mobile friendly. Try to use a grid (dynamic) rather than a table.

Reply:
Pretty sure fixed https://github.com/FRCTeam1987/teambroncobots.com/commit/c89c13e3c3bca1a6cbe3fb146e16d839c0ea25a0